<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgSketchOptions</class>
 <widget class="QWidget" name="WdgSketchOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>222</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>210</horstretch>
    <verstretch>60</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>60</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnminimumwidth="0,1">
   <item row="0" column="0">
    <widget class="QLabel" name="radiusLabel">
     <property name="text">
      <string>Radius</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KDoubleNumInput" name="radiusKNumInp">
     <property name="value">
      <double>64.000000000000000</double>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>512.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="sliderEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Line width</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KDoubleNumInput" name="lineWidthSPBox">
     <property name="value">
      <double>1.000000000000000</double>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="suffix">
      <string> px</string>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="sliderEnabled">
      <bool>true</bool>
     </property>
     <property name="exponentRatio">
      <double>1.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Offset scale</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KDoubleNumInput" name="offsetSPBox">
     <property name="value">
      <double>30.000000000000000</double>
     </property>
     <property name="maximum">
      <double>200.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="sliderEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Density</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KDoubleNumInput" name="densitySPBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="sliderEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="lowOpacityCHBox">
     <property name="text">
      <string>Force low opacity</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="connectionCHBox">
     <property name="text">
      <string>Paint connection line</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="magnetifyCHBox">
     <property name="text">
      <string>Magnetify</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
